docs/conf.py

```diff
@@ -36,34 +36,29 @@
 # -- General configuration ---------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
+# needs_sphinx = '1.0'
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = [
-    'sphinxcontrib.moderncmakedomain',
-    'sphinx.ext.autodoc',
-    'sphinx.ext.viewcode',
-    'sphinx_issues'
```

Comment:
Nitpick: I usually keep such list spread across multiple lines and alphabetically sorted to allow easier review.

Reply:
Black shouldn’t have been run on conf.py, we had decided to leave it unformatted. (I’m fine to format it, but not part of this PR, and it needs to be in-excluded in pre-commit. And adding a trailing comma would keep it per-line). So this file should be reverted.

Reply:
PS: Also a fan of making everything where order doesn't matter alphabetically sorted. :)